frontend/server/src/Controllers/Problem.php

```diff
@@ -47,7 +47,8 @@
  * @psalm-type Tag=array{name: string}
  * @psalm-type ProblemListCollectionPayload=array{levelTags: list<string>, problemCount: list<array{name: string, problems_per_tag: int}>, allTags: list<Tag>}
  * @psalm-type ProblemPrintDetailsPayload=array{details: ProblemDetails}
- * @psalm-type LibinteractiveGenPayload=array{error: array{description: null|string, field: null|string}|null, idl: null|string, language: null|string, name: null|string, os: null|string}
+ * @psalm-type LibinteractiveError=array{description: null|string, field: null|string}
```

Comment:
```suggestion
 * @psalm-type LibinteractiveError=array{description: string, field: string}
```

no hay necesidad de tener todo nulable. si ya estamos estableciendo el error, _debería_ tener ambos campos poblados.

para que psalm no se queje en L5807, se puede poner un error default, y como campo se puede usar el idl.

Reply:
Listo.